### PR TITLE
Merge CLU 1.0.0 release branch to main

### PR DESCRIPTION
**Pull request by heaths**
Because CLU 1.1 beta was meant to ship before CLU 1.0 GA due to features announced at MSBuild, it was originally in main while I created a release branch for CLU 1.0 GA to stabilize it; however, a decision was made to ship CLU as a DPG though it wasn't complete. As issues were found and the 1.0 GA date approached, I needed to switch gears to 1.0 GA while DPG was still taking updates. This is better done in main.

This PR is a literal copy of the release/clu-2022-05-01 branch I created above. Because we prefer a linear history, dealing with all the merge conflicts that would abound wasn't worth it. So I created this branch / PR by:

1. Deleting everything in _sdk/cognitivelanguage_.
2. Ran `git checkout release/clu-2022-05-01 -- sdk/cognitivelanguage` to copy all files from that branch's HEAD.
3. Ran `eng/scripts/CodeChecks.ps1 cognitivelanguage` to make sure generated code was up to date, despite having just recently taking the same autorest.csharp version into my previous release/clu-2022-05-01 branch.

Since history of my commits - which should have helpful descriptions - will be lost, I wanted to include a bit of the history here in case it's ever required for troublshooting.

### CODE REVIEW

File: sdk/cognitivelanguage/Azure.AI.Language.Conversations/CHANGELOG.md [hunk 1]
```diff
@@ -1,16 +1,23 @@
 # Release History
 
-## 1.1.0-beta.1 (Unreleased)
+## 1.0.0-beta.4 (Unreleased)
```
Comment (annelo-msft) [inline sdk/cognitivelanguage/Azure.AI.Language.Conversations/CHANGELOG.md:3]: Is it Beta and not GA?

**Review by annelo-msft — APPROVED** [review]
Approving to enable merge.

File: sdk/cognitivelanguage/Azure.AI.Language.Conversations/CHANGELOG.md (hunk 1 above)
Comment (heaths) [reply sdk/cognitivelanguage/Azure.AI.Language.Conversations/CHANGELOG.md:3]: I moved the beta out of main to release/clu-2022-05-15-preview. main is now GA; or, rather, pre-GA - I'll do a separate PR to make it final after I finish up another PR coming in for archboard feedback.

**Comment by heaths** [comment]
BTW, if anyone wants to check this without reviewing all the files - which @kristapratico and others have been doing as I made PRs against the old release/clu-2022-05-01 branch - in a clone of this repo just run:

```bash
git fetch upstream main
git fetch upstream release/clu-2022-05-01
git diff upstream/main..upstream/release/clu-2022-05-01 -- sdk/cognitivelanguage
```

You should see nothing. I even did this myself before pushing this branch to make sure I didn't miss anything.

**Pull request merged**